### PR TITLE
fix: limit length of `description` embed field

### PR DESCRIPTION
**Pull request by sgoudham**
Investigated the code as I was wasn't seeing an event for the payload included in this PR's fixture and I believe that it's because the PR body wasn't being limited on discord.

Figure we should move it to a "global" level but let me know if this is is a bad idea.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -23,6 +23,8 @@ const fn default_port() -> u16 {
     3000
 }
 
+const MAX_DESCRIPTION_LENGTH: usize = 640;
```
Comment (sgoudham) [inline src/main.rs:26]: We can change this if you want, I just wanted the channel to still be easily scrollable.

File: src/main.rs [hunk 2]
```diff
@@ -349,6 +345,14 @@ fn make_discord_message(e: &Event) -> anyhow::Result<Option<serde_json::Value>>
         return Ok(None);
     }
 
+    // Limit description length to make sure it doesn't make scrolling the Discord channel an
```
Comment (sgoudham) [inline src/main.rs:348]: I figure it's okay doing this check at a "global" level? I don't think there should be any perf impact but I would like to measure it going forward once this is more stable.

Comment (backwardspy) [reply src/main.rs:348]: i'd recommend doing this [in EmbedBuilder::description](https://github.com/catppuccin/rockdove/blob/35f265f87e675eea2389c2f3a336820027bff5a6/src/main.rs#L222-L225) instead - it should end up much cleaner!

File: src/main.rs (hunk 1 above)
Comment (backwardspy) [reply src/main.rs:26]: this is fine, the number i chose was based on measuring messages from the official github<>discord integration, but we can use anything we like.

**Review by backwardspy — CHANGES_REQUESTED** [review]
just a little improvement to the description length stuff and this is good to go

File: src/main.rs (hunk 2 above)
Comment (sgoudham) [reply src/main.rs:348]: Not sure how this escaped my mind but yup definitely, I'll get around to it tonight if you don't end up implementing it before me!

Comment (backwardspy) [reply src/main.rs]: had some other stuff to do (fix the line endings, update dependencies) so i moved the limits into EmbedBuilder and also limited the title & author name while i was in there.

**Comment by backwardspy** [comment]
i've commandeered this pull request

**Review by backwardspy — APPROVED** [review]

**Pull request merged**